package-lock.json

```diff
@@ -20,6 +20,33 @@
       "integrity": "sha1-kbR5JYinc4wl813W9jdSovh3YTU=",
       "dev": true
     },
+    "abstract-leveldown": {
```

Comment:
Should this file be included in this PR?

Reply:
package-lock is in the ext-json repository, so not including it causes the travis tests to fail.